std_msgs/Header header

float64[36] cov_estimate_best
float64[36] cov_estimate_5
float64[36] cov_estimate_10
float64[36] cov_estimate_20
float64[36] cov_mean

int32 resampled

int32 lines
int32 goals
int32 fb_points
int32 corners
int32 tcrossings
int32 crosses